api/test/wfl/system/v1_endpoint_test.clj

```diff
@@ -362,27 +362,29 @@
   [n workload]
   (try (endpoints/get-workflows workload)
        (catch Throwable x
-         (debug/trace x)
+         (debug/trace [n workload x])
          (when (> n 0)
            (try-to-get-workflows (dec n) workload)))))
 
 (defn ^:private summarize-workflows-in-workload
   "Summarize the workflows in `workload`."
   [workload]
-  (let [workflows (try-to-get-workflows 3 workload)]
-    {:count     (count workflows)
+  (let [workflows (try-to-get-workflows 3 workload)
+        statuses  (set (keep :status workflows))]
+    {:count     (count statuses)
+     :statuses  statuses
      :workload  workload
      :workflows workflows}))
 
 (deftest ^:parallel test-workflows-by-filters
   (testing "Get workflows by status"
-    (let [{:keys [workflows workload]}
+    (let [{:keys [statuses workflows workload]}
           (->> (endpoints/get-workloads)
+               (filter :finished)
```

Comment:
(Thinking out loud) I guess `:finished` implies that the workload probably has workflows.  We have some system test workloads that are created but never started (and thus never finished).

Reply:
I am not requesting changes.  Since the point of the test is to fetch workflows by filters, I was convincing myself that this probably won't lead you to grab 7 workloads with no workflows by accident or bad luck or whatever.  You could have workloads that are `:finished` with no workflows, but that's less likely.